FightPandemics/Views/Components/Buttons/SelectionSheetButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SelectionSheetButton.swift
+//  FightPandemics
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
```

Comment:
should it say FightPandemics where it says Scott?

Reply:
It's a Xcode generated comment from the original committed file I did when starting this.. we can delete all these on the files if we want, but it's just a comment generated from file creation.. do what you want 👍 as it makes no difference

Reply:
I understand. Ok, so we can to replace the files to "FightPandemics" as Copyright. Thanks Scott!